packages/moonstone/VirtualList/VirtualListBase.js

```diff
@@ -441,20 +520,12 @@ const VirtualListBaseFactory = (type) => {
 
 		jumpToSpottableItem = (keyCode, target) => {
 			const
-				{cbScrollTo, dataSize, isItemDisabled} = this.props,
+				{cbScrollTo, dataSize, isItemDisabled, wrap, wrapAnimated} = this.props,
+				{findSpottableExtent, findSpottableItem, getExtentIndex} = this,
```

Comment:
Please define variables from this.uiRef, then variable from this.

Reply:
Okay.